hack/get_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
i feel bad for nitpicking, but this file is actually new, not copied from kubernetes, right?

Reply:
Ouch I am dumb. I somehow assumed all copyrights in the repo are already attributed to kubevirt developers and just copy pasted. Let me fix it.